easybuild/easyconfigs/c/cdat/pycf-1.6.9-openblas.patch

```diff
@@ -0,0 +1,27 @@
+diff -ru pycf-1.6.9/setup.py pycf-1.6.9-edit/setup.py
```

Comment:
Please add patch description + link to upstream source or mention patch author (+ fix SHA256 for this patch file)